lib/elixir/lib/list.ex

```diff
@@ -1159,6 +1161,25 @@ defmodule List do
     end
   end
 
+  @doc """
+  Returns a new list subtracting `list2` from `list1`.
+  For each element in `list2` the first occurence in `list1` is deleted.
+
+  ## Examples
+
+      iex> List.subtract([1, 2, 3], [1, 2])
+      [3]
+
+      iex> List.subtract([1, 2, 3, 2, 1], [1, 2, 2])
+      [3, 1]
+
+  """
+  @doc since: "1.17.0"
+  @spec subtract(list, list) :: list
+  def subtract(list1, list2) when is_list(list1) and is_list(list2) do
+    :lists.subtract(list1, list2)
+  end
```

Comment:
```suggestion
  defdelegate subtract(list1, list2), to: :lists
```

Reply:
We should delegate to `:erlang.--` and it should be inline by the compiler (see elixir_rewrite.erl).

Reply:
done in https://github.com/elixir-lang/elixir/pull/13530/commits/f85ac966cf5600956f904214a97349b9f8552e3b